util/make_new_dif.py

```diff
@@ -71,10 +82,24 @@ def main():
         handle=handle,
     )
 
-    dif_file = args.output or dif_dir / 'dif_{}.h'.format(ip_snake)
-    with dif_file.open('w') as f:
+    header_file = args.output or dif_dir / 'dif_{}.h'.format(ip_snake)
+    with header_file.open('w') as f:
         f.write(header)
-    print('Template sucessfuly written to {}.'.format(str(dif_file)))
+
+    print('DIF header successfully written to {}.'.format(str(header_file)))
+
+    with checklist_template_file.open('r') as f:
+        markdown_template = Template(f.read())
+
+    checklist_file = header_file.with_suffix('.md')
+    with checklist_file.open('w') as f:
+        f.write(markdown_template.render(
+            ip_name=ip_snake,
+            dif_name=ip_snake,
+            display_name=periph_upper,
+        ))
```

Comment:
Would you mind adding a CLI flag to specify only generating one of the two? E.g. suppose I only want to generate the markdown file.

Reply:
oh so *this* is how we end up with every utility having five hundred different arguments.

Sure, but… we are allowed to be more opinionated for our tools, for instance, it's almost certainly not sensible to allow the template path to be configurable.

Reply:
Yeah so I removed `--template` and `--output` in favour of the script automatically picking up the right places, which seems like a better approach in the face of having multiple templates that may get instantiated to different locations.

Reply:
I actually use --output when doing renames to avoid having existing files get clobbered. =/

I'm ok with getting rid of --template but --output I really would prefer to keep (even if we change it so that it's --output_dir or whatever).

Reply:
Ok, I've kept it, but it is now a directory, not a filename.